plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/utils.ts

```diff
@@ -91,3 +91,14 @@ export function dimRatioToClass( ratio: number ) {
 		? null
 		: `has-background-dim-${ 10 * Math.round( ratio / 10 ) }`;
 }
+
+/**
+ * Convert long texts into short preview format with trailing dots.
+ *
+ * @param {string} text   store the the long string.
+ * @param {number} length preview text's length.
+ * @return {string} The truncated preview text.
+ */
+export function getLongTextPreview( text: string, length: number ): string {
```

Comment:
This code looks okay, but in WooCommerce we already have the `trimWords` function:

https://github.com/Kallyan01/woocommerce/blob/585311cd260a97010fbaca2a692766ff9694fef4/plugins/woocommerce/client/blocks/assets/js/utils/trim-words.ts#L37-L58

Could you try to re-use it?